lib/generators/install/install_generator.rb

```diff
@@ -6,6 +6,6 @@ def setup_initializer
     template 'mse_router_info.yml', 'config/mse_router_info.yml'
     copy_file 'microservices_engine.rb',
               'config/initializers/microservices_engine.rb'
-    generate 'migration', 'create_microservices_engine_connections', 'name:string', 'object:string', 'url:string'
+    generate 'migration', 'create_microservices_engine_connections name:string object:string url:string'
```

Comment:
We should move onto making a migration template instead, probably. This doesn't seem to work.